packages/actions/src/helpers/security.ts

```diff
@@ -0,0 +1,38 @@
+import fetch from "node-fetch"
+
+/**
+ * This function will return the number of public repos of a user
+ * @param user <string> The username of the user
+ * @param token <string> The token of the user
+ * @returns <number> The number of public repos
+ */
+const getNumberOfPublicRepos = async (user: string, token: string): Promise<number> => {
+    const response = await fetch(`https://api.github.com/users/${user}/repos`, {
+        headers: {
+            Authorization: `token ${token}`
```

Comment:
should be `authorization`?

Reply:
actually works either way, however will make sure they all have the same capitalization.